src/CommunityToolkit.Maui/Converters/BaseConverter.shared.cs

```diff
@@ -11,6 +11,11 @@ namespace CommunityToolkit.Maui.Converters;
 /// <typeparam name="TParam">Type of parameter</typeparam>
 public abstract class BaseConverter<TFrom, TTo, TParam> : ValueConverterExtension, ICommunityToolkitValueConverter
 {
+	/// <summary>
+	/// Default value to return when the value is null.
+	/// </summary>
+	public TTo? DefaultReturnType { get; set; } = default;
```

Comment:
I think the name is not correct. It should be DefaultReturnValue. Should we make it abstract or virtual?

Reply:
Good catch! Fixed the name to `DefaultReturnValue` ✅

I vote for `abstract` because it'll force contributors who add new converters to thoughtfully consider the `DefaultReturnValue`. This way, a new contributor can't accidentally overlook (and forget about) configuring the `DefaultReturnValue`